core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java

```diff
@@ -63,7 +65,9 @@
       ThreadFactoryUtils.build("block-worker-heartbeat-%d", true));
   private static final ExecutorService HEARTBEAT_CANCEL_POOL = Executors.newFixedThreadPool(5,
       ThreadFactoryUtils.build("block-worker-heartbeat-cancel-%d", true));
-
+  private static final WriteTier CLIENT_WRITE_TIER_POLICY =
```

Comment:
I think technically this is not a full policy but just the write tier? What is considered a "policy"?